fbw-common/src/systems/shared/src/popup.ts

```diff
@@ -77,6 +78,8 @@ export class PopUpDialog {
     callbackYes: () => void,
     callbackNo: () => void,
   ): void {
+    Coherent.trigger('UNFOCUS_INPUT_FIELD', uuidv4()); // Needed to mitigate an issue when ALT-TAB or using toggle free look
```

Comment:
Might make sense to move those to` _showPopUp` so it's effective for `showInformation` as well

Reply:
Good point, I'll consider moving it.